### ISSUE TITLE
Network path doesn't saved.

### ISSUE THREAD

**Issue by Cris--** [marijnkampf/Random-Photo-Screensaver#80]
Network path doesn't saved.

When i add "\\server\photos\son" and go back to tree view all fine.
I can see all custom network path.
Saving ok. Then close and reopen settings the custom path not saved anymore.  :(

RPS 4.5.10.1 (IE:11.0)
Windows 10 x64 Build 14393

**Comment by marijnkampf** [comment]
Please make sure there is no other version of RPS running. If it shows Read Only - RPS is already running in bottom left corner next to number of images your settings won't be saved. Easiest way to access settings is to press S whilst RPS is running.

**Comment by Cris--** [comment]
Nope, only one instance running! Saving seems successful, no error message or so.
